ext/commonmarker/_util.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+RUBY_MAJOR, RUBY_MINOR = RUBY_VERSION.split(".").collect(&:to_i)
+
+PACKAGE_ROOT_DIR = File.expand_path(File.join(File.dirname(__FILE__), "..", ".."))
+PACKAGE_EXT_DIR = File.join(PACKAGE_ROOT_DIR, "ext", "selma")
+
+OS = case os = RbConfig::CONFIG["host_os"].downcase
+when /linux/
+  # The official ruby-alpine Docker containers pre-build Ruby. As a result,
+  # Ruby doesn't know that it's on a musl-based platform. `ldd` is the
+  # a more reliable way to detect musl.
+  # See https://github.com/skylightio/skylight-ruby/issues/92
+  if ENV["SKYLIGHT_MUSL"] || %x(ldd --version 2>&1).include?("musl")
```

Comment:
We just merged in support for musl builds in rb-sys and could help out trying to iron it all out if you [join the Slack](https://join.slack.com/t/oxidize-rb/shared_invite/zt-16zv5tqte-Vi7WfzxCesdo2TqF_RYBCw)

Reply:
Heck yeah, will join for sure. I’ve been hacking together [Rust in Ruby](https://www.gjtorikian.com/posts/2017/08/15/wrapping-rust-in-ruby-to-parse-gql) for so long, I’m excited to see tooling and a community behind it!